lib/Elastica/Query.php

```diff
@@ -348,10 +348,6 @@ public function addScriptField($name, AbstractScript $script)
      */
     public function addAggregation(AbstractAggregation $agg)
     {
-        if (!array_key_exists('aggs', $this->_params)) {
```

Comment:
@ruflin, this is `redundant` - so remove it
http://sandbox.onlinephpfunctions.com/code/8dc8d94b0ccf05c77dffaf2c3b69bda702463f29